src/parser/parser.yy

```diff
@@ -625,6 +627,12 @@ expression
     | expression KW_NOT_ENDS_WITH expression {
         $$ = new RelationalExpression(Expression::Kind::kNotEndsWith, $1, $3);
     }
+    | expression KW_IS_NULL {
```

Comment:
directly use `KW_IS KW_NULL`

Reply:
Limited by bison grammar rules, the above method will cause parsing conflicts.

Reply:
**Query** : `yield not true is null` 
**Expected parsing result** : `yield not constExpr is null`->`yield not (constExpr is null)`->`yield (not (constExpr is null))`->`yield yield_column`

---
**Rule1** : expression KW_IS_NULL
**Rule2** : expression KW_IS KW_NULL

---
When the parser pointer of bison goes to the next token of `constExpr`, `Rule1` will reduce `constExpr KW_IS_NULL`, then reduce `not (constExpr KW_IS_NULL)`, but `Rule2` does not reduce `constExpr KW_IS`, but it can already reduce `not constExpr` by other rule, which is obviously an unexpected result.

---
For details about bison and LALR(1) grammar, please refer to [https://www.gnu.org/software/bison/](https://www.gnu.org/software/bison/)